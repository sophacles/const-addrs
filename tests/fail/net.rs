use const_addrs::net;

fn main() {
    let _ = net!("192.168.1.1/300");
    let _ = net!("2001:db8::32::23/129");
}
