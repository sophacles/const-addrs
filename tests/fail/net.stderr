error: invalid address: 192.168.1.1/300
 --> tests/fail/net.rs:4:18
  |
4 |     let _ = net!("192.168.1.1/300");
  |                  ^^^^^^^^^^^^^^^^^

error: invalid address: 2001:db8::32::23/129
 --> tests/fail/net.rs:5:18
  |
5 |     let _ = net!("2001:db8::32::23/129");
  |                  ^^^^^^^^^^^^^^^^^^^^^^
